python/ql/src/Functions/ModificationOfParameterWithDefault.ql

```diff
@@ -93,7 +93,47 @@ class Mutation extends TaintSink {
     }
 }
 
-from TaintedPathSource src, TaintedPathSink sink
-where src.flowsTo(sink)
+/** copying prevents modification of the default value */
```

Comment:
This could do with a bit more elaborate documentation. Something like
"By default, copying preserves most kinds of taint. However, for tracking mutable values, copying in fact _sanitizes_ the value being tracked, since mutating the copy does not affect the original."